cnapy/cellnetanalyzer.py

```diff
@@ -157,3 +158,29 @@ def read_config(self):
                 print("Could not find abs_tol in cnapy-config.txt")
         except NoSectionError:
             print("Could not find section cnapy-config in cnapy-config.txt")
+
+    def read_cobrapy_config(self):
+        ''' Try to read data from cobrapy-config.txt into appdata'''
+        config_parser = configparser.RawConfigParser()
+        try:
+            config_parser.read(self.appdata.cobrapy_conf_path)
+            try:
+                Configuration().solver = config_parser.get('cobrapy-config', 'solver')
```

Comment:
I would separate reading config and setting solver parameters.

Reply:
Probably not worth the trouble because Configuration() checks that a vaild and available solver is set.